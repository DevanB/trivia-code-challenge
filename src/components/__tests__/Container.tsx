import {render} from '@testing-library/react'
import * as React from 'react'
import {Container} from '../Container'

describe('<Container/>', () => {
  it('should render without crashing', () => {
    const {container} = render(<Container />)
    expect(container).toMatchSnapshot()
  })
})
