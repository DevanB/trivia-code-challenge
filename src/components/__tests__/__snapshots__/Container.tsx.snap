// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container/> should render without crashing 1`] = `
<div>
  <div
    class="Container-v7morq-0 gmvSeB"
  />
</div>
`;
